docker-compose.yaml

```diff
@@ -0,0 +1,36 @@
+version: "3.9"
+
+volumes:
+  app-logs:
+
+services:
+  am:
+    image: autometrics/am:latest
+    extra_hosts:
+      - host.docker.internal:host-gateway
+    ports:
+      - "6789:6789"
+      - "9090:9090"
+    container_name: am
+    command: "start http://otel-collector:9464/metrics"
+    environment:
+      - LISTEN_ADDRESS=0.0.0.0:6789
+    restart: unless-stopped
+    volumes:
+      - app-logs:/var/log
+  otel-collector:
+    image: otel/opentelemetry-collector-contrib:latest
+    container_name: otel-collector
+    command: ["--config=/etc/otel-collector-config.yaml"]
+    volumes:
+      - ./configs/otel-collector-config.yaml:/etc/otel-collector-config.yaml
+    ports:
+      - "4317:4317"
+      - "4318:4318"
+      - "8888:8888" # expose container metrics in prometheus format
+      - "9464:9464" # expose the collected metrics in prometheus format
```

Comment:
in the otel collector config, i added a prometheus exporter that exposes metrics on this port